src/types/types.ts

```diff
@@ -9,3 +9,11 @@ export interface ISignUpPayload {
   email: string;
   password: string;
 }
+
+export interface IFormValues {
+  name: string;
+  lastName: string;
+  email: string;
+  password: string;
+  password2: string;
```

Comment:
I think this should be called  passwordRepeat or passwordConfirm

Reply:
done